react/features/base/participants/middleware.js

```diff
@@ -127,6 +128,24 @@ MiddlewareRegistry.register(store => next => action => {
     return next(action);
 });
 
+// Syncs the redux state features/base/participants up with the redux state
```

Comment:
style: why not use multi-line comments?

Reply:
Let me understand this please. The style of comment:

````javascript
/**
 * Blah-blah
 */
````

would signify a jsdoc comment (in my understanding).

But what follows it is a function call, not a function definition. So I'm explaining what the "next line" which is a function call intends to implement and that (in my understanding) is not the above style of comment, it's a plain, non-special type of comment.

We're doing this weird mismatch of comment styles on `MiddlewareRegisry` and `ReducerRegistry` uses where the comment should actually be on the argument of the `register` call which is a function definitions. I presume your comment comes from there. While I don't like the weird mismatch of comment styles, I'll address your comment for the sake of consistency.

Reply:
OK. Will come in "Remove remote participants who are no longer of interest".

Reply:
Sorry I missed this. Generally, I prefer multi-line (/* ... */) comments for multi-line comments. But I know we are not 100% consistent here either.